docs: add more breaking changes about rsbuild 1.0 (#3245)
web-infra-dev/rsbuild: website/docs/en/guide/migration/rsbuild-0-x.mdx

});
```

## [Important] output.targets

:::tip
Rsbuild v1 removes the `output.targets` option and the target parameters of `source.alias` /`source.entry` and other configs, and instead configures it through `environments` to provide more flexible multi-environment config capabilities.

Compared with the original options, `environments` has a wider coverage and can perform differentiated config of more configs in multiple environments. For details, please refer to [Multiple-Environment Builds](/guide/advanced/environments).
:::

> Removed `output.targets` config, use `output.target` and `environments` config instead.

Reviewer: ```suggestion
Removed `output.targets` config, use [output.target](/config/output/target) and [environments](/config/environments) config instead.
```

  },
};
```

## Others

> Rsbuild 1.0 has made some adjustments and optimizations to APIs such as plugins and Dev Server.

Reviewer: ```suggestion
Rsbuild 1.0 has made some adjustments and optimizations to plugins API and dev server API.
```
